lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -23,7 +23,7 @@ def initialize(hosts, options)
       @logger = options[:logger]
 
       # Get AWS credentials
-      creds = load_credentials()
+      creds = ! options[:use_fog_credentials] ? Hash.new : load_credentials()
```

Comment:
Personally I prefer to work off positive conditions rather than negative conditions, so I'd be inclined to change this line to:

`creds = options[:use_fog_credentials] ?  load_credentials() : Hash.new`

Reply:
@ferglor Updated